### ISSUE TITLE
[Bug] Error output if first byte is "\n"

### ISSUE THREAD

**Issue by bvqhuynh** [TeamRocketIst/ctf-usb-keyboard-parser#4]
Hi team,

I found a small bug in your code is if the first byte in list bytes extract by tshark is '\n' (28), the code will display only '\n' than end process with nothing else data. I changed a little bit in your code, from  `\n` in line 41 to 0x28:['Enter\n','Enter\n'], and it work for me. Hope you can change it in your source code to help another guy.

Thanks.
Faceless

**Comment by ada-l0velace** [comment]
Thanks for reporting the bug, could you show me the input example so I can test it please ?

**Comment by bvqhuynh** [comment]
Hi, 

Sorry for late reply, here is the input data I tried:

00:00:28:00:00:00:00:00
00:00:1a:00:00:00:00:00
00:00:13:00:00:00:00:00
00:00:1a:00:00:00:00:00
00:00:0b:00:00:00:00:00
00:00:14:00:00:00:00:00
00:00:16:00:00:00:00:00
00:00:07:00:00:00:00:00
00:00:0b:00:00:00:00:00
00:00:0f:00:00:00:00:00
00:00:13:00:00:00:00:00
00:00:24:00:00:00:00:00
00:00:0b:00:00:00:00:00
00:00:1b:00:00:00:00:00
00:00:23:00:00:00:00:00
00:00:26:00:00:00:00:00

**Comment by ada-l0velace** [comment]
this is actually an interesting bug and is caused because the script is only saving data after a new line(enter) or the arrow keys up and down are pressed, since there wasn't an enter in the end the data wasn't being saved in the list.

I fixed the problem in a different way otherwise it would cause the rest of the tests to fail:
```python
...
if output != '' and output not in lines:
        lines[cursor_y] = output
return '\n'.join(lines)
```
I also had to modify the output of some tests because they were actually wrong since they actually didn't impact the flag reveal or important information I never noticed this was happening.

Can you share which ctf is this ? or this input isn't related to a ctf challenge?

**Comment by bvqhuynh** [comment]
Hi,

This challenge is `Stream` at VolgaCTF quals 2021; normally I used your script, but this payload make me weird and I found that bug.

**Comment by ada-l0velace** [comment]
> Hi,
> 
> This challenge is `Stream` at VolgaCTF quals 2021; normally I used your script, but this payload make me weird and I found that bug.

Thank you the bug was fixed in both [4744962](https://github.com/TeamRocketIst/ctf-usb-keyboard-parser/commit/4744962220b675f180373b4794e56441d4cc028a) and [7e3f00c](https://github.com/TeamRocketIst/ctf-usb-keyboard-parser/commit/7e3f00c6d037c176157e5e2762165f720a42ce95) commits , thanks for your contribution!
